sage-ios/SAGE/SAGE/SchoolOperations.swift

```diff
@@ -11,15 +11,23 @@ import AFNetworking
 
 class SchoolOperations {
 
-    static func loadSchools(completion: ((NSMutableArray) -> Void), failure: (String) -> Void){
-        let operationManager = BaseOperation.manager()
-        operationManager.GET(StringConstants.kEndpointSchool, parameters: nil, success: { (operation, data) -> Void in
-            
-            let schoolDict = data["schools"] as! NSMutableArray
-            completion(schoolDict)
-            }, failure: { (operation, error) -> Void in
+    static func loadSchools(completion: (([School]) -> Void), failure: (String) -> Void){
+        let manager = BaseOperation.manager()
+        manager.GET(StringConstants.kEndpointGetSchools, parameters: nil, success: { (operation, data) -> Void in
+            var schools = [School]()
+            let schoolArray = data["schools"] as! [AnyObject]
+            for schoolDict in schoolArray {
+                let school = School(propertyDictionary: schoolDict as! [String: AnyObject])
+                schools.append(school)
+            }
+            schools.sortInPlace({ (school1, school2) -> Bool in
+                school1.isBefore(school2)
+            })
```

Comment:
its weird cuz its all done by 1 query for checkins. for verified, we wanna sort by most recent start time. but for unverified we sort by most recent creation time. then again for verified i think the order will stay the same if we order by creation time